cms/tests/test_extensions.py

```diff
@@ -31,8 +31,9 @@ def test_register_extension(self):
         page_extension = self.get_page_extension_class()
 
         # register first time
-        extension_pool.register(page_extension)
+        register_extension_result = extension_pool.register(page_extension)
```

Comment:
That's what I meant. Just change the way the extension is registered in the test application, and remove the register part in the test, and we're done, no need for more explicit testing of the decorator implementation details